### PR TITLE
Display range of color selection for custom colors (#111)

### LINKED ISSUE #111: Display range of color Selection for the custom colors
So people can see what it looks like in the middle and the ends. 

Automatically detect minimum and maximum values

### PR DESCRIPTION
**Pull request by stanleykywu**
Adds new html with script for dynamic updating of gradient

question: I can't run the algorithm yet (working with Sarah next standup), are color changes saved correctly?

### CODE REVIEW

File: app/templates/pains_verify_and_coefficient_use.html [hunk 1]
```diff
@@ -1,120 +1,143 @@
+
+
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html]: Do we need these two new lines?

File: app/templates/pains_verify_and_coefficient_use.html [hunk 2]
```diff
@@ -1,120 +1,143 @@
...
 <div class="container">
-  <div class="text-center"> 
-    <h1> {{ title }}</h1>
-    <h3>Verify Pains Filter</h3> 
-
-    <form action="\verify_pains_by_error" method=POST enctype=multipart/form-data>
+<div class="text-center">
+   <h1> {{ title }}</h1>
+   <h3>Verify Pains Filter</h3>
+   <form action="\verify_pains_by_error" method=POST enctype=multipart/form-data>
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html]: I don't know that we need to change all of the formatting here.

File: app/templates/pains_verify_and_coefficient_use.html [hunk 3]
```diff
@@ -1,120 +1,143 @@
...
+               var color2 = document.querySelector(".color2"); 
+               var body = document.querySelector(".gradient"); 
+               
+               // Changing color for the gradient 
+               function changeGradient() { 
+                 body.style.background = 
+                       "linear-gradient(to right, "  
+                           + color1.value + ", "  
+                           + color2.value + ")"; 
+               }
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html]: Is there any way we could make this slightly smaller and next to the color selectors

File: app/templates/pains_verify_and_coefficient_use.html (hunk 2 above)
Comment (stanleykywu) [reply app/templates/pains_verify_and_coefficient_use.html]: VSCode changed some of the formatting, I have changed it back in latest commit

File: app/templates/pains_verify_and_coefficient_use.html (hunk 3 above)
Comment (stanleykywu) [reply app/templates/pains_verify_and_coefficient_use.html]: image

Is this better? (Also in latest commit)

File: app.py [hunk 4]
```diff
@@ -0,0 +1 @@
+from app import app
```
Comment (aislinblack) [inline app.py]: Did you mean to push this?

File: app/templates/pains_verify_and_coefficient_use.html (hunk 3 above)
Comment (aislinblack) [reply app/templates/pains_verify_and_coefficient_use.html]: Any chance we could cut the size of the gradient in half again? I just feel like it's super dominant lol

**Comment by stanleykywu** [comment]
Updated!

File: app/templates/pains_verify_and_coefficient_use.html [hunk 5]
```diff
@@ -56,14 +56,48 @@ <h3>Verify Pains Filter</h3>
   <br>
     <div class="text-left">
       <form action="\final_compounds" method=POST enctype=multipart/form-data>
-        <script id='include_mpo_script' include_mpo={{include_mpo}}>
-          include_mpo = document.getElementById('include_mpo_script').getAttribute('include_mpo');
-          if (include_mpo == "True") {
-            document.write('Mpo Score 0 Color: <input type="color" name="mpo0Color"/><br>Mpo Score 6 Color: <input type="color" name="mpo6Color"/><br>');
-          } else {
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html:63]: We still need this code for the minute while we use mpo score as a default option. Last year we allowed the input to be optional.

File: app/templates/pains_verify_and_coefficient_use.html [hunk 6]
```diff
@@ -56,14 +56,53 @@ <h3>Verify Pains Filter</h3>
...
+          <div class="parent">
+            <div style="margin-right:30px">
+              <b>Current Color for mpo0:</b> 
+              <input class="color1" type="color" name="mpo0color" value="#0000ff" /> 
+              <br>
+              <b>Current Color for mpo6:</b> 
+              <input class="color2" type="color" name="mpo6color" value="#add8e6" /> 
+            </div>
+            <div class="gradient" style="display: flex; justify-content: center; align-items: center;">
+              <div>Color Gradient</div>
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html:81]: I hate to be picky but now it's too small (lol truly a no win scenario)

File: app/templates/pains_verify_and_coefficient_use.html [hunk 7]
```diff
@@ -56,14 +56,53 @@ <h3>Verify Pains Filter</h3>
...
+                
+                color1.addEventListener("input", changeGradient); 
+                color2.addEventListener("input", changeGradient); 
+              </script>
+            </div>
+          </div> 
+        {% else %}
+          <b>Display Color</b>
+          <input type="color" name="mpo0Color"/>
+        {% endif %}
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html:103]: I'm getting an issue with running the algorithm. Lets look into this during the team portion of kickoff today!

**Review by aislinblack — APPROVED** [review]
Bug seems to be fixed and size looks great!

**Pull request merged**